lib/Installation/Partitioner/LibstorageNG/v4/_3/ClonePartitionsDialog.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for Expert Partitioner
```

Comment:
Let's write an specific summary for this page.

Reply:
A lot of documentation is currently missing, I've added brief description for this dialog.